### ISSUE TITLE
Pinia settings panel is not loaded immediatly when current page does not have any `useStore`

### ISSUE THREAD

**Issue by alexzhang1030** [vuejs/devtools#672]
Reproduction step:

- goto some page does not include any `useStore`
- open devtools pinia page (settings panel does not appear)
- goto some page include `useStore`
- refresh devtools
- settings panel appears

Record:

https://github.com/user-attachments/assets/ca32f8aa-fc05-4194-91c2-2020fb07e9fd

**Comment by webfansplz** [comment]
Yes, I noticed this, but I guess it has something to do with the design of the pinia devtools plugin, and maybe devtools v6 as well. But anyway, I think it makes sense to register pinia and let it come out. So if we confirm that it is the design problem of pinia devtools plugin, I think we can submit PR for it.

**Comment by alexzhang1030** [comment]
> Yes, I noticed this, but I guess it has something to do with the design of the pinia devtools plugin, and maybe devtools v6 as well. But anyway, I think it makes sense to register pinia and let it come out. So if we confirm that it is the design problem of pinia devtools plugin, I think we can submit issue or PR to pinia repo for it.

Pinia will register devtools once they have at least one store, see 

https://github.com/vuejs/pinia/blob/128f4352d2c18c0992cbab67b1a9261ff8a766ca/packages/pinia/src/store.ts#L731C8-L744

So plugin settings will update when somewhere has `useStore` calls.

But applet does not subscribe the plugin changes, should we support it?

**Comment by alexzhang1030** [comment]
Upstream issue, related https://github.com/vuejs/pinia/issues/2818
